### ISSUE TITLE
[Bug] Incompatible with Not Enough Trinket Slots

### ISSUE THREAD

**Issue by ConqAra** [TimVincii/Terrastorage#14]
### What happened?

I think it thinks [inventory tab's](https://modrinth.com/mod/inventory-tabs) player inventory is registered as a chest, so when opening my inventory it will always show the extra buttons on the sides when it's not suppose to!

### Terrastorage Version

Terrastorage-1.2.0+1.21-1.21.1

### Fabric API Version

fabric-api-0.109.0+1.21.1

### Minecraft Version

1.21.1

**Comment by ConqAra** [comment]
![2024-11-22_02 06 12](https://github.com/user-attachments/assets/5616f700-49b2-400e-a1b6-3ee31539d00c)
![2024-11-22_02 06 16](https://github.com/user-attachments/assets/9844ecbe-c099-441c-a21e-41d2c2530fae)
![2024-11-22_02 06 26](https://github.com/user-attachments/assets/2f8a05c0-b82c-434f-899c-47ce3856e7eb)
![2024-11-22_02 06 29](https://github.com/user-attachments/assets/7619ccd5-0589-4051-a2dd-23b957e9262e)

**Comment by TimVincii** [comment]
Hi! thanks for taking the time to submit a bug report.

This is quite the funny bug, I'm confused how Terrastorage even carries out renaming the player, as that makes no sense lmao.
Anyway, I tried replicating this on my end using just Terrastorage 1.2.0, Inventory Tabs 1.4.0, and Fabric API 0.109.0, but couldn't, the storage option buttons appear properly for storages with at least 27 slots, excluding the player's inventory.

This suggests something else, perhaps another mod or library, is modifying something related to how Terrastorage adds buttons to storage screens.
Could you provide a list of the mods you're using? Other than that - if you have can think of any other details you believe might be useful, please let me know.

**Comment by ConqAra** [comment]
here's the log with the modlist: [https://pastebin.com/6ij331kp](https://pastebin.com/6ij331kp)
I do remember it working too actually but I thought maybe i didn't notice it, maybe the recent mods I added a few days back a weird thing with it.

**Comment by ConqAra** [comment]
I removed my most recent mods I added and it fixed itself, I'm not sure what mod is doing this? 
![image](https://github.com/user-attachments/assets/0953a9cc-6510-45e2-a3fe-478ede0b8381)

**Comment by ConqAra** [comment]
OK, i found the incompatibility. it's [NETS](https://modrinth.com/datapack/nets). Seems like it has a very weird incompatibility with Terrastorage. Even though trinkets doesn't?

**Comment by TimVincii** [comment]
Thanks a lot for pinpointing which mod causes the issue! will take a look at it now.

**Comment by TimVincii** [comment]
Hi again! 

The issue was that NETS added enough slots to the player's inventory to make Terrastorage detect it as a storage screen, and thus the storage option buttons were added.

I already finished fixing this by improving the storage detection logic and will release the update today or tomorrow.

I’ll be closing this issue as resolved now. Once again, thank you very much for taking the time to report this bug and for going out of your way to provide screenshots, the mods list, and for identifying which mod specifically caused the issue, I really appreciate it!

Have a great day!
